fabric/patches/9_0001-add-proxy-support-for-docker-in-peer.patch

```diff
@@ -10,42 +10,42 @@ Signed-off-by: Michael Steiner <michael.steiner@intel.com>
  2 files changed, 52 insertions(+), 1 deletion(-)
 
 diff --git a/core/chaincode/platforms/util/utils.go b/core/chaincode/platforms/util/utils.go
-index 2e54e86ba..07fabf965 100644
+index 1d0562463..db7b29524 100644
 --- a/core/chaincode/platforms/util/utils.go
 +++ b/core/chaincode/platforms/util/utils.go
 @@ -10,6 +10,7 @@ import (
  	"bytes"
  	"fmt"
  	"io"
-+        "os"
++	"os"
  	"runtime"
  	"strings"
  
 @@ -52,6 +53,29 @@ type DockerBuildOptions struct {
  //      - OutputStream: A tarball of files that will be gathered from /chaincode/output
  //                      after successful execution of Cmd.
  //-------------------------------------------------------------------------------------------
-+func getProxyEnv()[]string {
-+        proxyEnv := []string{}
-+        proxyEnvVarNames := []string{
-+                "http_proxy",
-+                "https_proxy",
-+                "no_proxy",
-+                "HTTP_PROXY",
-+                "HTTPS_PROXY",
-+                "NO_PROXY",
-+        }
-+        for _, name := range proxyEnvVarNames {
-+                if val, found := os.LookupEnv(name); found {
-+                        proxyEnv = append(proxyEnv, name+"="+val)
-+                }
-+        }
-+        logger.Debugf("Util Proxy config used for docker: %v", proxyEnv)
-+        return proxyEnv
++func getProxyEnv() []string {
++	proxyEnv := []string{}
++	proxyEnvVarNames := []string{
++		"http_proxy",
++		"https_proxy",
++		"no_proxy",
++		"HTTP_PROXY",
++		"HTTPS_PROXY",
++		"NO_PROXY",
++	}
++	for _, name := range proxyEnvVarNames {
++		if val, found := os.LookupEnv(name); found {
++			proxyEnv = append(proxyEnv, name+"="+val)
++		}
++	}
++	logger.Debugf("Util Proxy config used for docker: %v", proxyEnv)
++	return proxyEnv
```

Comment:
did you use gofmt for this formatting?

Reply:
yep, the changes to the non-tlcc related patches are all just to properly `gofmt ..` so it passes `make linter`